sources/mongodb/helpers.py

```diff
@@ -37,40 +37,62 @@ def __init__(
         self.client = client
         self.collection = collection
         self.incremental = incremental
+
         if incremental:
             self.cursor_field = incremental.cursor_path
-            self.last_value = incremental.last_value
         else:
-            self.cursor_column = None
-            self.last_value = None
+            self.cursor_field = None
 
     @property
     def _filter_op(self) -> Dict[str, Any]:
-        if not self.incremental or not self.last_value:
+        """Build a filter operator.
+
+        Describes what logical statement to apply to which field.
+
+        Returns:
+            Dict[str, Any]: The filter operator.
+        """
+        if not self.incremental:
             return {}
-        if self.incremental.last_value_func is max:
-            return {self.cursor_field: {"$gte": self.last_value}}
-        elif self.incremental.last_value_func is min:
-            return {self.cursor_field: {"$lt": self.last_value}}
-        return {}
 
-    def load_documents(self) -> Iterator[TDataItem]:
-        cursor = self.collection.find(self._filter_op)
-        while docs_slice := list(islice(cursor, CHUNK_SIZE)):
-            yield map_nested_in_place(convert_mongo_objs, docs_slice)
+        if self.incremental.start_value:
```

Comment:
are tests really passing? because you removed `min` and `max` functions which reverse the range criteria! so if tests are passing please add them